src/hooks/useQuests.js

```diff
@@ -0,0 +1,8 @@
+import { QuestContext } from '../context/QuestsContext'
+import { useContext } from 'react'
+
+export default function UseQuests() {

```

Comment:
Hooks' practice is to name them starting in lower case. The upper case is usually reserved for components.